pkg/config/config.go

```diff
@@ -80,14 +80,23 @@ type ServerSecurityOptions struct {
 	// These are the Access-Control-Request-Headers that the server will respond to.
 	// By default, the server will allow Accept, Accept-Language, Content-Language, and Content-Type.
 	// DeprecatedUser this setting to add any additional headers which are needed
-	AllowedHeaders []string `json:"allowedHeaders"`
+	AllowedHeaders []string         `json:"allowedHeaders"`
+	RateLimit      RateLimitOptions `json:"rateLimitOptions"`
 }
 
 type SslOptions struct {
 	CertificateFile string `json:"certificateFile"`
 	KeyFile         string `json:"keyFile"`
 }
 
+// declare RateLimitConfig
+type RateLimitOptions struct {
+	Enabled           bool            `json:"enabled"`
+	RequestsPerSecond int             `json:"requestsPerSecond"`
+	BurstSize         int             `json:"burstSize"`
+	CleanupInterval   config.Duration `json:"cleanupInterval"`
+}
+
```

Comment:
This part needs docs for clarification. Would some one please kindly show me where to update the docs for configs ?

Thanks

Reply:
Resolved as inline docs is updated